tests/unit_and_functional/test_cli.py

```diff
@@ -206,11 +206,10 @@ def test_deployment_subcommand_exists():
 def test_deployment_test_locally_option_calls_run_workflow_handler(
```

Comment:
Rename to `test_deployment_run_locally_option_calls_run_workflow_handler`, because this is another p*ss poor name from me...

Reply:
It's just a typo, don't be so hard on yourself.

Reply:
It's never a good thing when you don't understand what it is you did a while back...